peekingduck/declarative_loader.py

```diff
@@ -78,22 +78,28 @@ def __init__(
 
             self.custom_nodes_dir = custom_nodes_dir
 
-    def _load_node_list(self, run_config_path: Path) -> List[str]:
+    def _load_node_list(self, run_config_path: Path) -> "NodeList":
```

Comment:
Does the typehint need to have double quotes, or can it be without it like in `runner.py`?

Reply:
The quotes are needed as NodeList is defined in the same script. More info [here](https://stackoverflow.com/a/33533514), the "Python <3.7: use a string" section in particular